src/main/java/org/opentripplanner/graph_builder/module/osm/OSMDatabase.java

```diff
@@ -211,11 +211,9 @@ public boolean isNodeBelongsToWay(Long nodeId) {
     public void addNode(OSMNode node) {
         if (node.isBikeRental()) {
             bikeRentalNodes.put(node.getId(), node);
-            return;
         }
         if (node.isBikeParking()) {
             bikeParkingNodes.put(node.getId(), node);
-            return;
         }
         if (!(waysNodeIds.contains(node.getId()) || areaNodeIds.contains(node.getId()) || node
```

Comment:
Could you add an `else if` where appropriate here?

Reply:
There is no need for an `else if`.

Is it preferred to have code style changes with functionality changes, or would it make better sense to have that in separate commits / pull requests?

Would reformating the file using the [intellij-code-style.xml](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/intellij-code-style.xml) be acceptable? Does that represent the accepted code style?

Reply:
We have some problem with getting the intellij-code-style.xml to work properly. Sometimes it do not make the desired line breaks. Especially it tend to leve do this:
```
    AType a = service.foo(argumentShouldBeOnNextLine,
        nextArgument,
        andSoOn
    );
```
the correct way is:

```
    AType a = service.foo(
        argumentShouldBeOnNextLine,
        nextArgument,
        andSoOn
    );
```
I use it on blocks of code, and sometimes on entire files - but a look over and fix the problem above.

There are settings for this, and I spent a day to get it to work, but sadly it does not. I think the underlying problem is the precedence of the format rules and the order they are applied.

I am very sorry this does not work. I am happy to adjust the code-style, if we can find a way to make this work and maintain readability. I want code in the same scope(e.g. method arguments) to be indented to the same level.  

When we have a code-style that works I would love to apply it on the entire Java code code base.